src/models/Filter.js

```diff
@@ -198,4 +198,27 @@ export default class Filter {
 
     return numFilters;
   }
+
+  toSearchParams() {
```

Comment:
Let's add the methods in the same way for consistency.  Filter.toSearchParams() or move the .fromSearchParams() inside the class?

Reply:
TIL that JS has a `static` keyword for just this purpose 😄 
I'll move `fromSearchParams()` inside as a static method. Thanks!